src/render.py

```diff
@@ -0,0 +1,30 @@
+import os
+from constant import BOARD_SIZE
+
+def get_marble_character(value):
+    return "◯" if value == 0 else "◉"
+    
+def print_board(board, clear_before_printing = True):
+    
+    if clear_before_printing:
+        os.system("clear")
+
+    # Used to detect when adding a line for drawing quarters.
+    HALF_BOARD_SIZE = (BOARD_SIZE / 2)
```

Comment:
who do not you add this in a constant file and instead of half_board_size you can use quarter_size